library/test/src/console.rs

```diff
@@ -147,7 +147,7 @@ pub fn list_tests_console(opts: &TestOpts, tests: Vec<TestDescAndFn>) -> io::Res
     let mut ntest = 0;
     let mut nbench = 0;
 
-    for test in filter_tests(&opts, tests) {
+    for test in filter_tests(&opts, tests).into_iter().rev() {
```

Comment:
I'm a little confused by the rev() here -- can we have a comment or something? It looks like it's mirroring a previously present reverse() call, but that also didn't have any comment.

It seems like if we want to reverse the test order, presumably that should happen inside the filtering?

Reply:
Tests were always sorted backwards, so that they can be popped from a `Vec` in alphabetical order. I wasn't sure if that decision was well-liked, but I guess by your comment other people find this as odd as I did. So I've adjusted to sorting in-order and using a `VecDeque` for `pop_front`.

Reply:
Hm, I don't mind the reversal I guess, but it seems like the kind of thing that should definitely have a comment. A VecDequeue also sounds okay, though.